lib/Clients/Service/Localise.php

```diff
@@ -76,27 +77,25 @@ public function export( array $args ) : string {
 			throw new Exception( 'Authenticate should be called first' );
 		}
 
+		$lock      = LockHandler::get_instance();
+		$proj_name = $args['__project_name'];
+
 		$url = sprintf(
 			'https://localise.biz/api/export/locale/%s.%s',
 			$args['locale'],
 			$args['ext']
 		);
-		unset( $args['locale'], $args['ext'] );
+		unset( $args['locale'], $args['ext'], $args['__project_name'] );
 
 		$url .= $this->get_query_string( $args );
 
 		$res = $this->client->request(
 			'GET',
 			$url,
-			[],
-			// TODO: incorporate If modified since
-			// 'headers' => [
-			// 	'If-Modified-Since' => $args[ 'Last-Modified' ],
-			// ]
+			$this->get_export_options( $lock, $proj_name )
 		);
 
-		// TODO: do something with this.
-		// var_dump( $res->getHeaders()['Last-Modified'][0] );
+		$lock['localise'][ $proj_name ]['Last-Modified'] = $res->getHeaders()['Last-Modified'][0];
```

Comment:
Não seria mais simples e da mesma forma eficaz, usar o timestamp da própria tradução para comparar?

Reply:
No. The timestamp needs to be exact to the second of what localise expects. In .po files there is a timestamp which has everything except the seconds so it doesn't work.